examine.lic

```diff
@@ -0,0 +1,25 @@
+class Examine
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'target', regex: /^[A-z\s\-]+$/i, variable: true, description: 'Object to examine' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    @target = args.target
```

Comment:
💡 I don't think this needs to be an instance var, it could just be an arg to `examine_object`